packages/authentication/index.d.ts

```diff
@@ -1,6 +1 @@
-// Copyright IBM Corp. 2013,2017. All Rights Reserved.
-// Node module: @loopback/authentication
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
-
-export * from './lib';
+export * from './src';
```

Comment:
Please revert all changes in all `index.d.ts` and `index.js` files. I don't mind discussing what's the right content to place there (e.g. I agree we should be enabling strict mode), but that should be done in a standalone pull request.